src/sentry/models/grouptagkey.py

```diff
@@ -23,8 +23,8 @@ class GroupTagKey(Model):
     """
     __core__ = False
 
-    project = FlexibleForeignKey('sentry.Project', null=True)
-    group = FlexibleForeignKey('sentry.Group')
+    project_id = BoundedPositiveIntegerField(null=True)
```

Comment:
These now need normal `db_index=True` on them. Since we don't want to lose an index.

Reply:
Good catch, these aren't on `GroupTagValue` either though, should I add them?

Reply:
Yeah, that's my bad then.

It technically shouldn't change anything since an earlier migration of adding the table would have created it correctly. Adding it here is effectively just bookkeeping.